backend/routes/interview.py

```
@@ -39,9 +232,97 @@ def answer():
+                except Exception as transcription_error:
+                    print(f"Transcription error: {transcription_error}")
+                    return jsonify({"error": f"Speech-to-text conversion failed: {str(transcription_error)}"}), 500
+            
+            # Clean up the temporary file (only if not storing for technical interviews)
+            import os
+            if os.path.exists(temp_path):
+                os.remove(temp_path)
+                
+        except Exception as e:
+            print(e)
+            return jsonify({"error": f"Speech-to-text conversion failed"}), 500
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)